toml/src/main/kotlin/org/rust/toml/crates/local/CratesLocalIndexService.kt

```diff
@@ -8,6 +8,8 @@ package org.rust.toml.crates.local
 import com.intellij.openapi.components.service
 
 interface CratesLocalIndexService {
+    fun isReady(): Boolean
```

Comment:
Let's avoid this method in public API. As I can see, we need it only for test purposes. Let's add it only in `Impl` class with `TestOnly` annotation if it's really required

Reply:
We discussed with @avrong that this method is a part of API because it will used later in inspections based on the crates index